### PR TITLE
Pass seed and batch_size to the dataloader functions

### PR DESCRIPTION
**Pull request by Grahams-Uncle**

### CODE REVIEW

**Comment by Grahams-Uncle** [comment]
fix #42

File: nbs/01_datasets.ipynb [hunk 1]
```diff
@@ -25,7 +25,16 @@
    "cell_type": "code",
```
Comment (BirkhoffG) [inline nbs/01_datasets.ipynb:25]: <p>You should make <code>batch_size</code> as an optional argument.</p><p> </p><p><code></code><code>python</p><p>batch_size: Optional[int] = None # batch_size is deprecated in </code>DataModuleConfigs<code></p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: nbs/04_learning.ipynb [hunk 2]
```diff
@@ -58,6 +58,7 @@
     "#| exporti\n",
```
Comment (BirkhoffG) [inline nbs/04_learning.ipynb:58]: <pre class="ql-syntax" spellcheck="false">Line #32.        train_loader = data_module.train_dataloader(next(keys), t_configs.batch_size)
</pre><p>You are passing a <code>PRNGKey</code> to the function, but it is supposed to be a seed number. Why? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by BirkhoffG — CHANGES_REQUESTED** [review]
See above comments

**Comment by Grahams-Uncle** [comment]
fix #39
